packages/tools/devtools/devtools-view/src/EditFlagHelper.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+
+/**
+ * Utility function to get the current Fluent UI theme to use.
+ * @returns Theme object of FluentUI to be used for dev tool
+ */
+export function getEditFlagToUse(): { edit: boolean } {
```

Comment:
This probably isn't needed. We will always set the context to the value we get from the `getSupportedFeatures.Message` message.